docs/sources/developers/plugins/data-frames.md

```diff
@@ -73,7 +73,7 @@ A data transformation is any function that accepts a data frame as input, and re
 
 A data frame with at least one time field is considered a _time series_.
 
-For more information on time series, refer to our [Introduction to time series](https://grafana.com/docs/grafana/latest/guides/timeseries/).
+For more information on time series, refer to our [Introduction to time series](https://grafana.com/docs/grafana/latest/getting-started/timeseries/#introduction-to-time-series).
```

Comment:
```suggestion
For more information on time series, refer to our [Introduction to time series]({{< relref "../../getting-started/timeseries.md" >}}).
```

This is an internal link, so we should the relref shortcode. That way, broken links like this get caught when building the docs.